packages/donor/src/components/AppHeader/AppHeader.tsx

```diff
@@ -78,6 +93,25 @@ export default function ButtonAppBar({
           </Grid>
         </Grid>
       </Toolbar>
+      <div>
+        <SideDrawer
+          isOpen={showSideBar}
+          close={() => setShowSideBar(false)}
+          navProps={{
+            navItemsProps: [
+              {
+                title: "החשבון שלי",
```

Comment:
The page title says פרופיל so let's be consistent

Reply:
הפרופיל שלי